src/components/form/EnzymeMultiSelect.jsx

```diff
@@ -10,7 +10,8 @@ export default function EnzymeMultiSelect({ setEnzymes }) {
   const [error, setError] = React.useState(false);
   const [waitingMessage, setWaitingMessage] = React.useState('retrieving enzyme list from server...');
   React.useEffect(() => {
-    const url = `${import.meta.env.VITE_REACT_APP_BACKEND_URL}restriction_enzyme_list`;
+    // Built like this in case trailing slash
+    const url = new URL('restriction_enzyme_list', import.meta.env.VITE_REACT_APP_BACKEND_URL).href;
```

Comment:
@NicolasCARPi ended up using this. I thought that double slashes would be the same thing, as in directory paths. Now works either way. Thanks for flagging it. I removed the docs mentioning the trailing slash.